<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEventsDlg</class>
 <widget class="QDialog" name="CEventsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget{
	background-color:#485464;
	padding-left:1px;
	padding-right:1px;
	outline:0px;
	border:0px;
}

QListWidget::item
{
	color:#cdcfd2;
	border-radius:5px;
	margin-top:1px;
	margin-bottom:1px;
    background-color:#cdcfd2;
}

QListWidget::Item:selected{
	color:#ffffff;
	border:3px solid #8dd1f4;
	background-color:#ffffff;
}

QListWidget::item:hover
{
	color:#ffffff;
    border:3px solid #8dd1f4;
	background-color:#ffffff;
   
}

QListWidget::item::selected:active
{ 
	color:#ffffff;
    border:3px solid #8dd1f4;
	background-color:#ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListWidget" name="EventListWidget"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Event List</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddEventBtn">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/add_device.png</normaloff>:/images/add_device.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveEventBtn">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DownloadFileBtn">
       <property name="text">
        <string>Download</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetEventBtn">
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/reset_event.png</normaloff>:/images/reset_event.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
